ge/org.osate.ge.errormodel/src/org/osate/ge/errormodel/ui/palette/CreateErrorTypePaletteCommand.java

```diff
@@ -35,7 +35,13 @@
 import org.osate.xtext.aadl2.errormodel.errorModel.ErrorModelFactory;
 import org.osate.xtext.aadl2.errormodel.errorModel.ErrorType;
 
+/**
+ * Palette command for creating error types.
```

Comment:
use @link ErrorType here

Reply:
That was actually intentional but I adjusted it and similar ones to be consistent with the others.